src/components/accordion/accordion.ts

```diff
@@ -28,6 +30,11 @@ export class Accordion extends LitElement {
       (node) => node instanceof AccordionItem
     ) as AccordionItem[];
     this._childItems.map((item, index) => {

```

Comment:
You might be able to simplify the handling of accordion item children by using the Lit query decorator: https://lit.dev/docs/api/decorators/#queryAssignedElements